app/controllers/sessions_controller.rb

```diff
@@ -14,7 +17,12 @@ def create
   end
 
   def destroy
-    log_out
+    log_out if logged_in?
     redirect_to root_url
   end
+
+  def add_product(product)
+    add_to_cart(product.name)
+    redirect_to '/home'
```

Comment:
As Cristi said, you should use home_path or home_url instead of the literal path.

Reply:
Sure thing, I've updated them👍